build.gradle

```diff
@@ -14,33 +21,34 @@ sourceCompatibility = 11
 targetCompatibility = 11
 
 ext {
-    grpcVersion = '1.32.1'
-    protobufVersion = '3.13.0'
-    serviceGeneratorVersion = '3.2.0'
-    grpcCommonVersion = '3.10.0'
+    grpcVersion = '1.57.2'
+    protobufVersion = '3.24.2'
+    serviceGeneratorVersion = '3.4.0'
+    grpcCommonVersion = '4.3.0-dev'
 
     genDir = file("${project.rootDir}/src/gen")
```

Comment:
I believe that you should correct this directory because the new `com.google.protobuf` plugin generates data in a different place. Please, check that the final JAR has META-INFO with services file